bl/evaluation/printer.go

```diff
@@ -79,16 +81,29 @@ func textOutput(results *EvaluationResults, invalidFiles []*validation.InvalidFi
 	return nil
 }
 
-func parseToPrinterWarnings(results *EvaluationResults, invalidFiles []*validation.InvalidFile, pwd string, k8sVersion string) ([]printer.Warning, error) {
-	var warnings = []printer.Warning{}
+func parseToPrinterWarnings(results *EvaluationResults, invalidYamlFiles []*validation.InvalidYamlFile, invalidK8sFiles []*validation.InvalidK8sFile, pwd string, k8sVersion string) ([]printer.Warning, []printer.Warning, []printer.Warning, error) {
+	var invalidYamlWarnings = []printer.Warning{}
+	var invalidK8sWarnings = []printer.Warning{}
+	var failedEvaluationWarnings = []printer.Warning{}
 
-	for _, invalidFile := range invalidFiles {
-		warnings = append(warnings, printer.Warning{
+	for _, invalidFile := range invalidYamlFiles {
+		invalidYamlWarnings = append(invalidYamlWarnings, printer.Warning{
```

Comment:
it's good to split the yaml and k8s validations into different sets but they should still be under the same warnings object which is per file,  that way it will be easier to iterate over it later